src/gg.regression.unity.bots/Runtime/Scripts/GenericBots/Experimental/GameSpeedup.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using UnityEngine;
+
+namespace RegressionGames.GenericBots.Experimental
+{
+    /// <summary>
+    /// Configures the game to run as fast as possible for faster training/search.
+    /// The speedup is active as soon as this object is constructed.
+    /// When the Dispose() method is invoked, the speedup is disabled.
+    /// The user of this class needs to regularly invoke the Update() method to ensure the speedup
+    /// is retained.
+    /// </summary>
+    public class GameSpeedup : IDisposable
```

Comment:
@svolokh Make sure to test this out with project steelline.  They use timed animations of the cards shuffling into the game and I ran into timing issues with those in the past recording bot segments.

Reply:
I tested it out and it doesn't seem to break with the time scaling, but I did notice that with the more performance-intensive game the time scaling results in a very huge deltaTime that isn't great for training the bots. So I've reduced the default training scale to something that is probably more reasonable for most games (from 20 -> 3). It could be worth exploring dynamically adjusting the timeScale based on unscaledDeltaTime in the future.